rules/gcp/impact_gcp_storage_bucket_deleted.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/09/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a Google Cloud Platform (GCP) storage bucket is deleted. An adversary may delete a storage bucket in
+order to disrupt their target's business operations.
+"""
```

Comment:
Consider adding FP info, maybe something similar to below.
```
false_positives = [
    """
    Storage bucket deletions may be done by a system or network administrator. Verify whether the user email,
    resource name, and/or hostname should be making changes in your environment. Bucket deletions from unfamiliar users or
    hosts should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
    """,
]
```

Reply:
Consider it done